libs/journeys/ui/src/components/StepFooter/StepFooter.tsx

```diff
@@ -0,0 +1,94 @@
+import Box from '@mui/material/Box'
+import Stack from '@mui/material/Stack'
+import Typography from '@mui/material/Typography'
+
+import { ReactElement } from 'react'
+import { useJourney } from '../../libs/JourneyProvider'
+import { getJourneyRTL } from '../../libs/rtl'
+
+export function StepFooter(): ReactElement {
+  const { journey } = useJourney()
+  const { rtl } = getJourneyRTL(journey)
+
+  return (
+    <Box
+      className="swiper-no-swiping"
```

Comment:
```suggestion
      className="swiper-no-swiping"
```
he's just trying to make a living...